source/include/microapp/cs_MicroappProtocol.h

```diff
@@ -82,6 +83,10 @@ class MicroappProtocol { //: public EventListener {
 		 */
 		int _coskip;
 
+		/**
+		 * Store i2c data
+		 */
+		CircularBuffer<uint8_t>* _i2c_data;
```

Comment:
```suggestion
		CircularBuffer<uint8_t>* _i2cData;
```

Reply:
Fixed in https://github.com/crownstone/bluenet/commit/cf20c97dfe77cce02d2a80555f843335b0e7fa04. Renamed, because is general callback buffer.